### ISSUE TITLE
Icon Widget Improvements

### ISSUE THREAD

**Issue by seshelby** [elementor/elementor#9662]
When adding an icon wrapped in a link it currently results in an empty link as far as a screen reader is concerned. This is a big deal in the web accessibility world. Please add a text field under link settings to allow someone to enter an aria-label for the link. Some would argue that a title is more appropriate but either way one or the other should be included.

## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by shilo-ey** [comment]
Hi @seshelby 

Thanks for your feedback,

We are currently working on a broader solution for this issue (regarding links attributes)


For now I'm closing this issue, and this will be released in the future versions of Elementor.

Thanks
